src/System.Private.Uri/tests/ExtendedFuctionalTests/System.Private.Uri.ExtendedFunctional.Tests.csproj

```diff
@@ -7,6 +7,7 @@
     <OutputType>Library</OutputType>
     <AssemblyName>System.Private.Uri.Tests</AssemblyName>
     <NugetTargetMoniker>.NETStandard,Version=v1.5</NugetTargetMoniker>
+    <NoWarn>$(NoWarn);1685</NoWarn>
```

Comment:
This warning is because we added mscorlib targetting pack reference, so now Task<> exists in both mscorlib and System.Runtime. This will work for now, but we have to eventually remove this and make the project target TestTFM netcoreapp1.1 to get the right assets out of System.Runtime.pkgproj